crates/kitsune_p2p/direct/src/types/kdhash.rs

```diff
@@ -26,6 +26,12 @@ pub trait KdHashExt: Sized {
     /// convert from kitsune op hash
     fn from_kitsune_op_hash(op_hash: &KitsuneOpHash) -> Self;
 
+    /// convert to kitsune op hash
```

Comment:
Looks like the comment is about an OpHash while this is a Basis?
Same below

Reply:
```suggestion
    /// convert to kitsune basis
```